src/handlers/cache_control_handler.cr

```diff
@@ -0,0 +1,9 @@
+class CacheControlHandler
+  include HTTP::Handler
+
+  def call(context)
+    context.response.headers["Cache-Control"] = "public, s-maxage=#{180.days.to_i}, maxage=#{90.days.to_i}"
+    context.response.headers["Expires"] = 1.month.from_now.to_s("%a, %d %b %Y %H:%M:%S %z")
```

Comment:
I'm nervous about this because this sets cache control for everything, even the HTML (if I understand this correctly). I think this should only be for static assets, but HTML should not be cached (or be cached for a very short period of time). That way when we make updates people see the latest versions

Reply:
I think if I move this after the RouteHandler, then that should work.

Reply:
Ahh yeah I think so too. Can you test on the review app to make sure the header is only present for static assets just to make sure?

Reply:
What do you mean by "review app"? Like write the spec in this app? Or you're saying just test that in some other app?

Reply:
Oh I mean the Heroku review app that gets auto-deployed. Looks like it isn't deployed though so I'll make it now